exercises/common-lisp-1-a/README.md

```diff
@@ -0,0 +1,59 @@
+## Pokémon Battle
+
+As an aspiring [Pokémon][pokemon] trainer, you're looking to get some battle
+experience so you can hone your skills. The only problem is, you don't have any
+Pokémon! Being the resourceful trainer you are, you decide to write a simple
+computer program that will allow you to simulate Pokémon battles so you can be
+prepared for the real deal.
+
+### Making Pokémon
+
+In this simplified simulator, every Pokémon has a name, type, attack stat, and
+some HP (hit-points). You should define a function `make-pokemon` for
```

Comment:
Would it be possible to define a stub function for `make-pokemon`? This would help the student get started and is also how the other tracks define their stubs.

Reply:
We could define a stub for it if that's what's best. The reason I avoided it is that there are technically solutions that won't need to explicitly define that function. Using something like `defstruct` will automatically define that function and some others.

I left it out thinking it would encourage some more diverse solutions, but I can add it if you would like!

Reply:
Hmmm, that is a good argument. Let me think on this!

Reply:
@iHiD Any thoughts on this?

Reply:
Good idea, but I think @TheLostLambda made a good argument to leave it as it is. Obviously if this was v3 I'd be hard the other way, but for the research I think allowing multiple possible solutions is the most critical variable.

Reply:
Okay! Let's approve this then :)